lib/src/arithmetic/float_fixed.dart

```diff
@@ -0,0 +1,104 @@
+// Copyright (C) 2023-2024 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// floating_fixed.dart
+// Transform floating point input into fixed point output.
+//
+// 2024 September 25
+// Author: Soner Yaldiz <soner.yaldiz@intel.com>
+
+import 'package:rohd/rohd.dart';
+import 'package:rohd_hcl/rohd_hcl.dart';
+
+/// [FloatToFixedConverter] converts a floating point input to
+/// a signed fixed-point following Q notation (Qm.n format) as introduced by
+/// (Texas Instruments)[https://www.ti.com/lit/ug/spru565b/spru565b.pdf].
+/// Infinities and NaN's are not supported. Conversion is lossless.
+/// Separator bit sep = bias + mantissa - 1
+/// fixed[2*sep] contains sign
+/// fixed[2*sep-1:sep] contains fractional part
+/// fixed[sep-1:0] contains fractional part
+class FloatToFixedConverter extends Module {
+  /// Output port [fixed]
+  Logic get fixed => output('fixed');
```

Comment:
You could make this return a `FixedPoint`, something like `late final FixedPoint fixed = FixedPoint(...)..gets(output('fixed'))`